doc/README.rst

```diff
@@ -3,52 +3,60 @@
 The doc folder
 ==============
 
-The documentation for NEST is contained in this folder. We are currently working
-on improving the documentation and making updates. However, if you notice
-something out of place, you can `submit an issue <https://nest.github.io/nest-simulator/development_workflow#reporting-bugs-and-issues>`_
-in our GitHub repository.
+This folder contains the static parts of the documentation for NEST,
+as well as the infrastructure and tools for extracting documentation
+dynamically from the source code and building a unified HTML site
+using Sphinx.
 
-Build the docs locally
------------------------
+For each of the releases, the fully built documentation is available
+at https://nest-simulator.readthedocs.io/
+
+Build the user documentation locally
+------------------------------------
 
-You will need the NEST source code in your project::
+If you intend to work on the documentation, or if you want to obtain a
+local version of it for other reasons, you can build the documentation
+by simply running
 
+::
 
-    cd nest-simulator/doc/
+    make html
```

Comment:
This won't work any more, one now needs to run cmake first.

Reply:
Agreed. That's why the next lines say:
> in the build directory of NEST (i.e. the directory where you ran ``cmake``).

Reply:
Ah, good! But assuming that readers will be eager an try things as soon as they read them, I'd suggest to move the sentence up. Github does't let me create a multiline suggestion here, but like this:
```
by simply running in the NEST build directory (where you ran ``cmake``)

::
    make html

To install ...

Reply:
Agreed. Fixed in 3a94c83e3.